### PR TITLE
Add posibility turn of mysql backup for specific nodes,

### PR DESCRIPTION
**Pull request by avlasov-mos-de**
fix rsync creating remote directories,  add import vars revquired for loading cassandra service_grains.
List of related PRs coming later

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

**Comment by avlasov-mos-de** [comment]
Document explaining purpose and manual steps done https://docs.google.com/a/mirantis.com/document/d/1bi2m3OZ6v2ax7Xn_A8TpmAH-0S2eDSN5aDZYU-0I9I0/edit?usp=sharing

Related PRs:
https://github.com/Mirantis/mk2x-cookiecutter-reclass-model/pull/93
https://github.com/Mirantis/reclass-system-salt-model/pull/270
https://github.com/salt-formulas/salt-formula-backupninja/pull/1
https://github.com/salt-formulas/salt-formula-opencontrail/pull/19
https://github.com/salt-formulas/salt-formula-maas/pull/16

File: backupninja/map.jinja [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{%- from "linux/map.jinja" import system with context %}
```
Comment (fpytloun) [inline backupninja/map.jinja]: This should not be needed, why did you add it?

Comment (avlasov-mos-de) [reply backupninja/map.jinja]: I have parametrized meta/backupninja.ymk for contrail formula(check contrail formula PR) so when we load it service_grains jinja should be able to render it.

Comment (fpytloun) [reply backupninja/map.jinja]: Yes, but grains should be used instead.

**Comment by jiribroulik** [comment]
Why are you not using xtrabackup for galera/mysql backups?

**Comment by maestropandy** [comment]
Provide access to through https://docs.google.com/a/mirantis.com/document/d/1bi2m3OZ6v2ax7Xn_A8TpmAH-0S2eDSN5aDZYU-0I9I0/edit?usp=sharing to have glance

**Comment by slimakcz** [comment]
changes in backupninja/client.sls are really useful, because I have the same issue. I using xtrabackup for galera/mysql backups and for moving data to backup server I would like to use backupninja. I disabled backupninja for mysql in reclass model, but mysql handler was still generated.
Similar issue is with backup cassandra on network nodes, because I enabled backupninja, disable mysql backup, but I have mysql definition in reclass model, thus backupninja preparing mysql handler, but missing admin user/password (which is correct, I don't need this data on ntw node).

**Review by epcim — APPROVED** [review]
lgtm, an alternative to xtrabackup
